casaos-fix-docker-api-version/run.sh

```
@@ -368,12 +368,22 @@ verify_docker_api_version() {
     return 0
   else
     echo "⚠ WARNING: Docker API version is $api_version"
-    echo "Expected: 1.47 for CasaOS compatibility"
+    echo "Expected: 1.47 for CasaOS compatibility (Docker 28.0.x)"
     echo ""
-    echo "This might indicate:"
-    echo "  - The Docker package installation didn't work properly"
-    echo "  - A different Docker binary is being used"
-    echo "  - The Docker daemon didn't restart with the new version"
+    echo "Current version: $api_version"
+    if [[ "$api_version" > "1.47" ]]; then
```

Shellcheck (0.11.0)

[error] Decimals are not supported. Either use integers only, or use bc or awk to compare.

(SC2072)